src/application-base/baseStyles.js

```diff
@@ -5,6 +5,8 @@ if (process.env.NODE_ENV !== 'production') {
   // Check to ensure dir attribute is set on html element
   if (!document.documentElement.hasAttribute('dir')) {
     // eslint-disable-next-line
-    console.warn('The html element is missing the dir attribute. For terra directionality based styles to render correctly, add dir="ltr" or dir="rtl" to the html element.');
+    console.warn('The html element is missing the dir attribute. ' + 
```

Comment:
should this be a console.error instead?

Reply:
Given that it can wreck certain components hard, I think it should be logged as an error. Could also go through the Logger we have vs. standard console.

Reply:
do we want this error filtered from production? that seems to be the main difference between the Logger and standard console

Reply:
The initial implementation of this did, at least. How about this, I'll remove the explicit env check in this file, switch to the logger that already does an env check, and keep it a warning to ensure it gets filtered out in prod. If something sneaks through to prod, an error in the console ain't gonna help much 🤷‍♂️

Reply:
sounds good to me!